### PR TITLE
fix: Optimize article description

### PR DESCRIPTION
**Pull request by qiuming520**

### CODE REVIEW

File: kosmos/docs/v0.2.0/tutorials/design-of-exec-and-log.md [hunk 1]
```diff
@@ -238,5 +238,5 @@ By default, InternalDNS has the highest priority.
 ![EXE_Log_3.png](img/EXE_Log_3.png)
```
Comment (ONE7live) [inline kosmos/docs/v0.2.0/tutorials/design-of-exec-and-log.md]: Wouldn't it be impressive if we used console output here?

Comment (ONE7live) [reply kosmos/docs/v0.2.0/tutorials/design-of-exec-and-log.md]: [root@kosmos-control-cluster ~]# kubectl get nodes -owide
NAME               STATUS   ROLES                         AGE   VERSION   INTERNAL-IP    EXTERNAL-IP   OS-IMAGE                KERNEL-VERSION               CONTAINER-RUNTIME
kosmos-control-1   Ready    worker                        65d   v1.21.5   192.168.0.1    <none>        CentOS Linux 7 (Core)   3.10.0-957.27.2.el7.x86_64   docker://20.10.8
kosmos-control-2   Ready    control-plane,master,worker   65d   v1.21.5   192.168.0.2    <none>        CentOS Linux 7 (Core)   3.10.0-957.27.2.el7.x86_64   docker://20.10.8
kosmos-cluster7    Ready    agent                         20d   v1.21.5   192.168.0.3    <none>        <unknown>               <unknown>                    <unknown>

File: kosmos/docs/v0.2.0/tutorials/design-of-exec-and-log.md [hunk 2]
```diff
@@ -1,25 +1,25 @@
...
+Since the overall architecture of `kubectl exec` and `kubectl log` is the same, we will use `kubectl exec` as a sample to introduce the overall architecture. 
 The following diagram illustrates the overall design architecture.
 
 ![EXE Log_Arch.png](img/EXE_Log_Arch.png)
 
 ### Background Knowledge
-First, let's sort out how exec is implemented in k8s. 
-The leaf-cluster part of the architecture diagram is a representation of the exec function of a general cluster. 
+First, let's study how `kubectl exec` is implemented in Kubernetes. 
+The _**leaf-cluster**_ part of the architecture diagram is a representation of the `kubectl exec` function of a native cluster.
```
Comment (OrangeBao) [inline kosmos/docs/v0.2.0/tutorials/design-of-exec-and-log.md:19]: How about changing  ```First, let's study how `kubectl exec` is implemented in Kubernetes. ``` to ```First, let's explore how to implement "kubectl exec" in Kubernetes.```?

Comment (qiuming520) [reply kosmos/docs/v0.2.0/tutorials/design-of-exec-and-log.md:19]: yes, i have done

**Pull request merged**